docs/api-guide.md

```diff
@@ -746,13 +803,12 @@ Delete delete =
 transaction.mutate(Arrays.asList(put, delete));
 ```
 
-#### Use a default namespace for CRUD operations
+#### Default namespace for CRUD operations
 
-A default namespace for all the CRUD operations can be set with a property of the ScalarDB configuration.
-If you would like to use this setting with ScalarDB server, it needs to be set on the client-side configuration.
+A default namespace for all CRUD operations can be set by using a property in the ScalarDB configuration. If you would like to use this setting with ScalarDB Server, you will need to configure the setting on the client-side configuration.
```

Comment:
I think we need to change this sentence for ScalarDB 3.9 and later docs since ScalarDB Server has been deprecated in ScalarDB 3.9 and later. 

In 3.9 and later docs, can we delete this sentence? Or should we reword it to mention ScalarDB Cluster instead?

Reply:
We can delete this sentence. Thanks.